master/internal/config.go

```diff
@@ -72,8 +73,9 @@ type Config struct {
 	Security              SecurityConfig                    `json:"security"`
 	CheckpointStorage     CheckpointStorageConfig           `json:"checkpoint_storage"`
 	TaskContainerDefaults model.TaskContainerDefaultsConfig `json:"task_container_defaults"`
-	HTTPPort              int                               `json:"http_port"`
-	HTTPSPort             int                               `json:"https_port"`
+	Port                  int                               `json:"port"`
+	HTTPPort              int                               `json:"http_port"`  // deprecated
+	HTTPSPort             int                               `json:"https_port"` // deprecated
```

Comment:
blocking: I would recommend we handle this case better, or remove

- we could try to translate them, but it's pretty confusing since how would we handle the case we're given both
- we could try to get both and provide a good error message
- we could just remove these fields

Reply:
I will just remove these fields.

Reply:
sounds good -- let's make sure we add `BREAKING_CHANGE` to the PR title (and make sure it shows up in the squashed commit)